astrobee/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ install(DIRECTORY launch
 
 # Mark resource files for installation
 install(DIRECTORY resources
-  DESTINATION lib
+  DESTINATION share/${PROJECT_NAME}
```

Comment:
need to change this such that we can find the resources folder from the launchfile